### ISSUE TITLE
Train only re-id branch

### ISSUE THREAD

**Issue by Smirkey** [ifzhang/FairMOT#337]
Hi,

Is there a straightforward way to train only the re-id branch, i.e take the boxes from the training data instead of the predicted boxes ? Also, is it a good idea to take object centers from training data instead of Learning them ?

Best,
Smirkey

**Comment by Smirkey** [comment]
One easy way is to change the loss to only take into account reid loss, not clean at all but it works

**Comment by ifzhang** [comment]
Yes, you can only train with reid loss. However, I find that the reid performance is better when training with both the detection loss and reid loss because the detection loss can help the reid task like guiding the reid branch to the bbox.

**Comment by Smirkey** [comment]
Thanks for the answer :)
Yes i understand that, and indeed it seems that training only reid is suboptimal. I will try to train only reid+heatmap, as I only need the encoding part!

**Comment by Smirkey** [comment]
results: meh
